contrib/test/ci/vars.yml

```diff
@@ -150,6 +150,11 @@ kata_skip_image_tests:
   - 'test "image pull and list by manifest list tag"'
   - 'test "image pull and list by manifest list and individual digest"'
   - 'test "image pull and list by individual and manifest list digest"'
+  - 'test "run container with memory_limit_in_bytes -1"'
+  - 'test "run container with memory_limit_in_bytes 12.5MiB"'
+  - 'test "run container with container_min_memory 17.5MiB"'
```

Comment:
Can we skip only the failing test? WDYT?

Reply:
Which one is it? Why only a single one would fail, but not all of them for Kata? :thinking:

Reply:
AFAIK only this test fails `run container with container_min_memory 17.5MiB"`

Reply:
For kata, the runtime settings written in the test cases are not used, but the kata runtime is used instead. (the settings are specified in vars.yml)

In the settings for kata, it doesn't set `container_min_memory`  explicitly so it will be 12MiB (default).

In the 17.5 MiB test, normally it is expected to be failed because the default runtime   `container_min_memory` is supposed to have 17.5 MiB but the container's memory limit is 12MiB (12MiB < 17.5 MiB).
However for kata, kata's `container_min_memory` is 12MiB and it succeeds in creating the container, which caused the test failure.

In my opinion, the other tests just happen to have the same result in both non-kata and kata, so it should be removed from the kata tests.
Also it's confusing because we might think the runtime set in the test cases are used although they are not used.